usb_port: "/dev/ttyUSB0"
baud_rate: 9600
poll_rate: 1
